StripeCore/StripeCore/Source/Connections Bindings/ElementsContext.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  ElementsContext.swift
+//  StripeCore
+//
+//  Created by Mat Schmid on 2024-09-23.
+//
+
+import Foundation
+
+@_spi(STP) public struct ElementsContext {
```

Comment:
nit: `ElementsContext` is a bit vauge, can we either add a doc comment about what/where this is being populated from e.g. elements/session or rename to ElementsSessionContext